test/evergreen.yml

```diff
@@ -3743,6 +3743,39 @@ buildvariants:
     - name: ".stress-test-4"
     - name: format-abort-recovery-stress-test
 
+- name: ubuntu2004-stress-tests-arm64
+  display_name: Ubuntu 20.04 Stress tests (ARM64)
+  run_on:
+  - ubuntu2004-arm64-large
+  expansions:
+    # Reduce the number of parallel operations, this is to prevent OOM killer interrupting the
+    # format sanitizer stress test.
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` - 4" | bc)
```

Comment:
Would halving the num of CPU's be more portable than reducing the num by 4? We have an example in the `ubuntu1804-ppc` build variant.

Reply:
Fixed